sys/include/net/ieee802154/radio.h

```diff
@@ -255,6 +282,7 @@ typedef struct {
      */
     uint8_t rssi;
     uint8_t lqi;    /**< LQI of the received frame */
+    uint8_t flags;  /**< flags as defined above */
```

Comment:
I'm wondering if we need this flag. Shouldn't be enough to have the CRC_ERROR event?

Reply:
I think in general it might be uneeded, I used in OpenWSN because of its internal logic:

- on a valid or invalid CRC a callback is called then `radio_getReceivedFrame` is called that then calls `ieee802154_radio_read` and so I used the `flags` to see if there was an invalid CRC

But if there is no other use case for this I can set a global variable for this and remove this extra `flags` variable. I'll do this since there is not other use currently.

Reply:
Addressed.

Reply:
perfect!